src/MongoDB.Driver.Core/Core/Authentication/ScramSha1Authenticator.cs

```diff
@@ -16,18 +16,21 @@
 using System;
 using System.Collections.Generic;
 using System.IO;
+using System.Runtime.InteropServices.ComTypes;
 using System.Security.Cryptography;
 using System.Text;
 using MongoDB.Bson.IO;
+using MongoDB.Driver.Core.Authentication.Vendored;
 using MongoDB.Driver.Core.Connections;
 using MongoDB.Driver.Core.Misc;
+using Rfc2898DeriveBytes = MongoDB.Driver.Core.Authentication.Vendored.Rfc2898DeriveBytes;
```

Comment:
As per our discussion on Slack this is because we want to use OUR vendored version of Rfc2898DeriveBytes instead of the one in .NET Framework.

Let's put a blank line to separate this using statement from the others (it serves a completely different purpose) and add a comment, something like:

    // use our vendored version of Rfc2898DerivedBytes instead of the standard one
    using Rfc2898DeriveBytes = MongoDB.Driver.Core.Authentication.Vendored.Rfc2898DeriveBytes;

Reply:
Sure, setting off the using statement from the rest of the using statement with an extra new line sounds like a good idea. As for the comment, I feel like I already have a comment to the effect? I can change the wording if you'd like.

Reply:
Added line break to ScramSha1Authenticator. Added both a line break and the additional comment to ScramSha256Authenticator

Reply:
Additionally, changed the wording to be closer to your format.

Reply:
My comment above was written *before* you had added any comment to the file. The comment you added to the file is fine.

Reply:
Sorry, I thought I pushed a version of the file with a comment on it before your comment, but perhaps it was just a timing mismatch--or another bit a wonkiness that seems to be plaguing the PRs for 2171! Apologies for the confusion--I think we're on the same page though :)